content/en/docs/home/contribute/localization.md

```diff
@@ -38,12 +38,17 @@ Source files reside in the `/docs/` directory.
 
 ### Repository
 
-A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-cn](https://github.com/kubernetes/kubernetes-docs-cn).
+A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-zh](https://github.com/kubernetes/kubernetes-docs-zh).
 
 {{< note >}}
 **Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
 {{< /note >}}
 
+#### Repository structure
+
+Each l10n repository must have branches for the different Kubernetes documentation release versions, matching the branches in the main [kubernetes/website](https://github.com/kubernetes/website) documentation repository. For example, the kubernetes/website `release-1.10` branch (https://github.com/kubernetes/website/tree/release-1.10) has a corresponding branch in the kubernetes/kubernetes-docs-zh repository (https://github.com/kubernetes/kubernetes-docs-zh/tree/release-1.10). These version branches keep track of the differences in the documentation between Kubernetes versions.
```

Comment:
Good ideas, but when we publish `release-XXX` branch  in `en` docs, who can help publish related `release-XXX` in `cn` docs? And this rules write here unnecessary

Reply:
It is important to mention this here. Before, all of the changes for different versions were going into the same branch in kubernetes/kubernetes-docs-zh. My plan is to have a doc sprint at the Shanghai KubeCon later this year to address the translation workflow issues.